tools/copydir.sh

```diff
@@ -96,5 +96,6 @@ fi
 
 cp -a "${src}" "${dest}" || \
   { echo "Failed to create link: $dest" ; rm -rf ${dest} ; exit 1 ; }
+touch "${src}"
```

Comment:
why?

Reply:
Just want to make sure include/arch etc targets are newer than .clean_context in dirlink stage. To avoid link more times in build. It seems link.sh/copydir.sh scripts only used in Makefile.unix dirlinks.

Reply:
i guess these belong to Makefile if any.

Reply:
Well, I'll update them into Makefile later.

Reply:
Done